tests/by-util/test_rmdir.rs

```diff
@@ -174,17 +174,12 @@ fn test_verbose_nested_failure() {
 #[cfg(unix)]
 #[test]
 fn test_rmdir_ignore_nonempty_no_permissions() {
-    use std::fs;
-
     let (at, mut ucmd) = at_and_ucmd!();
 
     // We make the *parent* dir read-only to prevent deleting the dir in it.
     at.mkdir_all("dir/ect/ory");
     at.touch("dir/ect/ory/file");
-    let dir_ect = at.plus("dir/ect");
-    let mut perms = fs::metadata(&dir_ect).unwrap().permissions();
-    perms.set_readonly(true);
-    fs::set_permissions(&dir_ect, perms.clone()).unwrap();
```

Comment:
Cool stuff, what was the clippy check for that?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#permissions_set_readonly_false

Apparently `set_readonly(false)` is equivalent to `chmod a+w`. It doesn't matter too much for our test suite, but I figured it was easy enough to change the test.